apps/marketplace/components/SellerAssessment/SellerAssessmentCriteriaStage.js

```diff
@@ -20,6 +20,35 @@ const getCriteriaNeeded = (criteriaNeeded, priceMaximum, maxDailyRate) => {
 const getCriteriaAllowed = (criteriaNeeded, priceMaximum, maxDailyRate) =>
   getCriteriaNeeded(criteriaNeeded, priceMaximum, maxDailyRate) + 2
 
+const getMinimumMessage = (criteriaNeeded, essentialCriteria) => {
+  if (essentialCriteria.length > 0) {
+    return (
+      <span>
+        You must select at least {criteriaNeeded - essentialCriteria.length}{' '}
+        <strong> &apos;Other criteria&apos;</strong>
+      </span>
+    )
+  }
+
+  return (
+    <span>
+      You must submit evidence for at least {criteriaNeeded} {criteriaNeeded === 1 ? 'criterion' : 'criteria'}
+    </span>
+  )
+}
+
+const getMaximumMessage = (criteriaAllowed, criteriaNeeded, essentialCriteria) => {
+  if (essentialCriteria.length > 0) {
+    return (
+      <span>
+        You must select a maximum of {criteriaNeeded} <strong> &apos;Other criteria&apos;</strong>
```

Comment:
Should this message be similar to the non-essential message (eg. You cannot submit evidence for more 3 'Other criteria')?
```suggestion
        You must select a maximum of {criteriaNeeded} <strong>&apos;Other criteria&apos;</strong>
```

Reply:
I believe for the message of this one, it should have been 'You can only select a maximum of **X 'Other Criteria'**